docs/appliance-config.md

```diff
@@ -0,0 +1,19 @@
+# Appliance Config
+* To generate a configuration `yaml`, please refer to [Generate a template of the appliance config](./user-guide.md#Generate-a-template-of-the-appliance-config)
+
+
+| Name                       | Default Value                  | Optional | Type    | Description                                                                                                                                                                            |
+|----------------------------|--------------------------------|----------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| apiVersion                 |                                | No       | enum    | The configuration version that is currently supported by the appliance. options: `v1beta1`.                                                                                            |
+| kind                       |                                | No       | enum    | The configuration kind that is currently supported by the appliance. options: `ApplianceConfig`.                                                                                       |
```

Comment:
This would actually stay constant for this yaml, it's simply the kind of the object. Let's change it to something like:
Type: string
Description: The configuration kind - 'ApplianceConfig'

Reply:
Done